03/app.js

```diff
@@ -11,4 +11,33 @@ const buttonSettings = {
         color: '#444'
     },
     text: 'Click me!',
-}
\ No newline at end of file
+}
+
+//dodanie buttona wewnątrz section
+const parentElement = document.querySelector('.parent-for-button');
+const newButton = document.createElement('button');
+parentElement.appendChild(newButton);
+console.log(newButton);
+
+// dodawanie atrybutów - pętla for in
+// 1 attr (obiekt) 
+// 2 css (obiekt)
+// 3 text (string)
+for(const elAttr in buttonSettings.attr
+    ) {
+    if(elAttr === 'className') {
+        newButton.className = buttonSettings.attr[elAttr];
+    } else {
+        newButton.setAttribute(elAttr, buttonSettings.attr[elAttr]);
+    }    
```

Comment:
Gdybyśmy użyli zapisu w formie `newButton[elAttr] = buttonSettings.attr[elAttr]` nie musielibyśmy używać warunku - dlatego wydaje się to lepsze rozwiązanie.

Reply:
ok